dlt/normalize/normalize.py

```diff
@@ -354,6 +354,12 @@ def spool_files(
         self.load_storage.new_packages.save_schema_updates(
             load_id, merge_schema_updates(schema_updates)
         )
+        # self.load_storage.new_packages.save_dropped_tables(
+        #     load_id, self.normalize_storage.extracted_packages.load_dropped_tables(load_id)
+        # )
+        self.load_storage.new_packages.save_load_package_state(
```

Comment:
OK! so we do not move the state from extract package to normalize package? maybe that should be a part of the normalize_storage

Reply:
this one got fixed! If you take the newest devel!